tests/aead_tests.rs

```diff
@@ -64,10 +75,11 @@ fn test_aead(aead_alg: &'static aead::Algorithm, file_path: &str) {
         let plaintext = test_case.consume_bytes("IN");
         let ad = test_case.consume_bytes("AD");
         let mut ct = test_case.consume_bytes("CT");
-        let tag = test_case.consume_bytes("TAG");
+        let mut tag = test_case.consume_bytes("TAG");
         let error = test_case.consume_optional_string("FAILS");
 
         let tag_len = aead_alg.tag_len();
+        tag.truncate(tag_len);
```

Comment:
Why truncate `tag` here instead of just changing the input file data to have the right length?

One of the reasons I want to use the Chromium test vectors is precisely to make sure we're chopping of the tag in the right place. That is, they give us confidence that chopping off the end of the 128-bit tag to get a 96-bit tag is the right thing to do.

Reply:
In fact, I copied [the behaviors from the Chromium](https://chromium.googlesource.com/chromium/src/+/master/net/quic/core/crypto/aes_128_gcm_12_decrypter_test.cc#261), I guess they want to keep a full tag.